_posts/platform/getting-started/2000-01-01-getting-started-with-elk.md

```diff
@@ -149,6 +149,21 @@ Logstash has created the unicorn index which can now be requested:
 The result of the above search contains a document having with a field `name`
 set to `Alenala` and a field `message` set to `Hi!`.
 
+### Custom configuration for Logstash
+
+In the logstash configuration repository, clone from
+[Scalingo/logstash-boilerplate](https://github.com/Scalingo/logstash-boilerplate),
+a config directory is present. All the files in this folder will be copied in the
+logstash configuration directory at runtime, allowing you to customize exactly
+how your want logstash to run.
```

Comment:
I find this paragraph hard to understand. Moreover, we already talk about cloning our boilerplate on line 41. Here is a rephrase suggestion, WDYT?

```suggestion
The cloned boilerplate used to deploy your application contains a `config` directory. 
All the files in this folder will be copied in the Logstash configuration directory at runtime, 
allowing you to customize exactly how your want Logstash to run.
```